data_collection/gazette/spiders/rn_macaiba.py

```diff
@@ -0,0 +1,41 @@
+import datetime as dt
+import re
+
+import dateparser
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class RnMacaibaSpider(BaseGazetteSpider):
+    name = "rn_macaiba"
+    TERRITORY_ID = "2407104"
+    allowed_domains = ["macaiba.rn.gov.br"]
+    start_urls = ["https://macaiba.rn.gov.br/servicos/diario-oficial/"]
+    start_date = dt.date(2013, 8, 9)
+
+    def parse(self, response):
+        for edition in response.xpath("//li[@class='link_item']"):
```

Comment:
Sugiro utilizar seletor css aqui. Pra lidar com classes normalmente css é melhor pois o xpath dessa maneira exige que a classe seja apenas `link_item`. Se o elemento passar a ter outra classe além de `link_item` esse código já quebraria.
```suggestion
        for edition in response.css("li.link_item"):
```

Reply:
Pra complementar acho que podemos realizar a seleção diretamente no item que temos interesse. Assim evitaria replicar o `a` nas seleções abaixo
```suggestion
        for edition in response.css("li.link_item a"):
```